lua/core/info.lua

```diff
@@ -134,36 +123,42 @@ function M.toggle_popup(ft)
   }
   vim.list_extend(buf_lines, header)
 
+  local client = u.get_active_client_by_ft(ft)
+  local client_enabled_caps = require("lsp").get_ls_capabilities(client.id)
+  local num_caps = vim.tbl_count(client_enabled_caps)
   local lsp_info = {
     "Associated language-server: " .. client.name,
-    indent .. "* Active: " .. tostring(is_client_active) .. ", id: " .. tostring(client.id),
+    indent .. "* Active: " .. tostring(not client.is_stopped) .. ", id: " .. tostring(client.id),
     indent .. "* Formatting support: " .. tostring(client.resolved_capabilities.document_formatting),
     indent .. "* Capabilities list: " .. table.concat(vim.list_slice(client_enabled_caps, 1, num_caps / 2), ", "),
     indent .. indent .. indent .. table.concat(vim.list_slice(client_enabled_caps, ((num_caps / 2) + 1)), ", "),
     "",
   }
   vim.list_extend(buf_lines, lsp_info)
 
+  local null_ls_providers = null_ls_handler.registered_providers_name(ft)
   local null_ls_info = {
     "Configured providers: " .. table.concat(null_ls_providers, "  , ") .. "  ",
     "",
   }
   vim.list_extend(buf_lines, null_ls_info)
 
-  local missing_formatters_status
+  local missing_formatters = null_ls_handler.langs[ft].errors.formatters
```

Comment:
`null_ls_handler.langs[ft]` can never be `nil`, right?

Reply:
I assume that [this](https://github.com/LunarVim/LunarVim/pull/1277/files#diff-61e47b385a127e61dcc5c97b396900ce66b93bf502422b58dcf30f5ba0566a09R86) should be called first